sssom/io.py

```diff
@@ -360,6 +360,8 @@ def filter_file(input: str, output: TextIO, **kwargs) -> MappingSetDataFrame:
             for exp in v[1:]:
                 query += " OR "
                 query += k + " LIKE '" + exp + "') "
+        else:
+            query += ") "
```

Comment:
this is new, I assume there was no test for the `else` case?

Reply:
This was missing and became apparent when only 1 value to a param was given for e.g. `--subject_id ABCD:1234` instead of multiple `--subject_id ABCD:1234 --subject_id DEFG:4567`. The missing `)` that's added in the code above threw an error. There was no test to check this.